pkg/system/phase4_configuring.go

```diff
@@ -1281,12 +1341,23 @@ func (r *Reconciler) ReconcileDefaultBucketClass() error {
 		return nil
 	}
 
-	r.DefaultBucketClass.Spec.PlacementPolicy = &nbv1.PlacementPolicy{
-		Tiers: []nbv1.Tier{{
-			BackingStores: []nbv1.BackingStoreName{
-				r.DefaultBackingStore.Name,
+	if !util.IsFusionHCIWithScale() {
```

Comment:
I prefer to check if we have DefaultBackingStore or if we have DefaultNamespaceStore, as in the future we might support more default namespacestores...

Reply:
yes. We don't need to handle HCI separately in bucket class.
in bucket class we should see: if we have default backingstore set -> it should point to it.
if not, if we have default namespacestore set -> it should point to it.
the logic on what to create was already handled in the default namepsace/backingstore logic.

Reply:
done.